timeseries/tests/unittests/models/test_ensemble.py

```diff
@@ -46,6 +48,13 @@ def _fit(self, predictions_per_window, data_per_window, model_scores=None, time_
     def _predict(self, data, **kwargs):
         return PREDICTIONS_FOR_DUMMY_TS_DATAFRAME
 
+    def remap_base_models(self, model_refit_map: Dict[str, str]) -> None:
+        pass
+
+    @property
+    def model_names(self) -> List[str]:
+        return list(PREDICTIONS_FOR_DUMMY_TS_DATAFRAME.keys())
```

Comment:
This looks a bit suspicious to me. If I understand correctly, `PREDICTIONS_FOR_DUMMY_TS_DATAFRAME` is a `pd.DataFrame`, so `list(PREDICTIONS_FOR_DUMMY_TS_DATAFRAME.keys())` is equivalent to `list(PREDICTIONS_FOR_DUMMY_TS_DATAFRAME.columns)`, which is the same as `["mean", "0.1", "0.2", ..., "0.9"]`. Is this intended?

Reply:
ha. asleep at the wheel again!